src/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -1013,6 +1013,7 @@ public static void MiscellaneousTest()
             Thread.SpinWait(0);
             Thread.SpinWait(1);
             Thread.Yield();
+            Thread.GetCurrentProcessorId();
```

Comment:
Maybe a test that validates it falls within some expected range, e.g. 0 to Environment.ProcessorCount?

Reply:
(although I guess since we use the managed thread ID as a fallback, it's hard to make any real statement about what the result here will be)

Reply:
Not negative, at least?

Reply:
Fixed.